client/state/reader/follows/schema.js

```diff
@@ -0,0 +1,22 @@
+export const items = {
+	type: 'object',
+	patternProperties: {
+		//be careful to escape regexes properly
+		'.+': {
```

Comment:
i don't love this but getting the set of things correct was ... fun. urls can have lots of weird things in them.